flink-runtime/src/test/java/org/apache/flink/runtime/testutils/ZooKeeperTestUtils.java

```diff
@@ -37,6 +41,33 @@ public class ZooKeeperTestUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(ZooKeeperTestUtils.class);
 
+    /**
+     * Creates a new {@link TestingServer}, setting additional configuration properties for
+     * stability purposes.
+     */
+    public static TestingServer createZookeeperTestingServer() throws Exception {
```

Comment:
```suggestion
    public static TestingServer createAndStartZookeeperTestingServer() throws Exception {
```
Just as a proposal to make it more obvious what the method does...